gobblin-service/src/test/java/org/apache/gobblin/service/modules/core/GobblinServiceRedirectTest.java

```diff
@@ -90,8 +90,8 @@
   private static final String TEST_SOURCE_NAME = "testSource";
   private static final String TEST_SINK_NAME = "testSink";
 
-  private static final String PORT1 = "1000";
-  private static final String PORT2 = "2000";
+  private static final String PORT1 = "10000";
```

Comment:
You can checkout PortUtils class that provides a method for handing out random unused ports. Static port values run the risk of port conflicts.